src/catch2/internal/catch_stringref.hpp

```diff
@@ -31,6 +31,8 @@ namespace Catch {
 
     public: // construction
         constexpr StringRef() noexcept = default;
+        constexpr StringRef(StringRef const&) noexcept = default;
+        constexpr StringRef& operator=(StringRef const&) noexcept = default;
```

Comment:
Why?

(Also, this disable moves. While moves aren't optimized for StringRef, I think this could be surprising in the future when StringRef reports to be nonmoveable under some type trait).

Reply:
Sigh. True. I just wanted to make it explicit that the class has copy operations, so that the `StringRef( std::string const& stdString )` overload is not selected for copy (which I am frankly not sure would be impossible if the `operator std::string` weren't `explicit`).

I'll remove it.